stdlib/public/Platform/Platform.swift

```diff
@@ -351,11 +351,6 @@ public func ioctl(
 public func fork() -> Int32 {
   fatalError("unavailable function can't be called")
 }
-
-@available(*, unavailable, message: "Please use threads or posix_spawn*()")
-public func vfork() -> Int32 {
```

Comment:
Now that you two have brought it up, I'm inclined to leave it in. It's a small thing that does improve the user experience.